controller/controller_server/utils.py

```diff
@@ -491,7 +490,7 @@ def get_node_id_info(node_id):
     elif len(split_node) == 2:
         hostname, nvme_nqn = split_node
     else:
-        raise array_errors.HostNotFoundError(node_id)
+        raise ValidationException(messages.wrong_format_message.format("node id"))
```

Comment:
I think this should be performed as part of a function like `validate_publish_volume_request`, instead of here

Reply:
and what will be raised here?

Reply:
`ValueError`/`return None`/whatever

Reply:
`ValueError` seems more convenient for the caller than `None`

Reply:
done